README.md

```diff
@@ -54,10 +63,14 @@ If you simply want to try out making playlists from a folder containing songs,
 [this example](https://github.com/Polochon-street/bliss-rs/blob/master/examples/playlist.rs)
 contains all you need. Usage:
 
-        cargo run --features=serde --release --example=playlist /path/to/folder /path/to/first/song
+        cargo run --release --example=playlist --features=serde /path/to/folder /path/to/first/song
 
 Don't forget the `--release` flag!
 
+To use Symphonia instead of FFmpeg to decode songs, use the symphonia-all feature flag instead:
+
+        cargo run --release --example=playlist --no-default-features --features=serde,aubio-static,symphonia,symphonia-all /path/to/folder /path/to/first/song
```

Comment:
`symphonia-all` automatically pulls in `symphonia`, so having both is a little redundant